pymc_experimental/model_builder.py

```diff
@@ -131,9 +131,15 @@ def create_sample_input():
         >>>       'chains': 1,
         >>>       'target_accept': 0.95,
         >>>    }
-        >>>    return data, model_config, sampler_config
```

Comment:
I think this should stay.

Reply:
oh, my bad. I was quite sure I added the Return section without deleting the example

Reply:
it's back again